src/main/assembly/dist/docs/examples/dcx-gml/example1.xml

```diff
@@ -65,7 +65,51 @@
             </LineString>
         </dcx-gml:spatial>
         
-        <!--  -->
+        <!-- box example -->
+        <dcx-gml:spatial>
+            <boundedBy xmlns="http://www.opengis.net/gml">
+                <Envelope srsName="http://www.opengis.net/def/crs/EPSG/0/28992">
+                    <lowerCorner>4 2</lowerCorner>
+                    <upperCorner>3 1</upperCorner>
+                </Envelope>
+            </boundedBy>
+        </dcx-gml:spatial>
+
+        <!-- polygon example -->
+        <dcx-gml:spatial>
+            <Polygon xmlns="http://www.opengis.net/gml">
+                <exterior>
+                    <LinearRing>
```

Comment:
I am only a bit worried that 'LinearRing' is not the only way to specify this in GML and that we have to support all the other options when converting to EMD. But we need to check this.

Reply:
Currently the GML scheme says:

```xml
<complexType name="PolygonType">
   <complexContent>
      <extension base="gml:AbstractSurfaceType">
         <sequence>
            <element ref="gml:exterior" minOccurs="0"/>
            <element ref="gml:interior" minOccurs="0" maxOccurs="unbounded"/>
         </sequence>
      </extension>
   </complexContent>
</complexType>
<element name="Polygon"
         type="gml:PolygonType"
         substitutionGroup="gml:_Surface"/>
<element name="exterior" type="gml:AbstractRingPropertyType"/>

<complexType name="AbstractRingPropertyType">
   <sequence>
      <element ref="gml:LinearRing"/>
   </sequence>
</complexType>
```

So, I guess there is only one kind: `LinearRing`.

Reply:
Nice!